cfgov/paying_for_college/models/search.py

```diff
@@ -0,0 +1,44 @@
+from paying_for_college.documents import SchoolDocument
+
+
+UNSAFE_CHARACTERS = [
+    '#', '%', ';', '^', '~', '`', '|',
+    '<', '>', '[', ']', '{', '}', '\\'
+]
+
+
+def make_safe(term):
```

Comment:
Don't we have a similar function for `ask-cfpb`? We should try to reuse them.

Reply:
Yes. I moved `UNSAFE_CHARACTERS` and `make_safe` to `elasticsearch_helpers.py` to be reused by `ask-cfpb`